src/clj/dashboard_clj/components/websocket.clj

```diff
@@ -67,3 +81,129 @@
                          :handler           client-ev-handler
                          :webserver-adapter webserver-adapter
                          :options           options}))
+
+
+(comment
```

Comment:
should probably add a "real" comment describing what/why this is here

Reply:
added